src/Presentation/RestApi/Controller/v1/HelloController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Presentation\RestApi\Controller\v1;
+
+use SprykerSdk\Sdk\Presentation\RestApi\Executor\Task\RestApiTaskExecutor;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+class HelloController
+{
+    /**
+     * @var \SprykerSdk\Sdk\Presentation\RestApi\Executor\Task\RestApiTaskExecutor
+     */
+    protected RestApiTaskExecutor $apiTaskExecutor;
+
+    /**
+     * @param \SprykerSdk\Sdk\Presentation\RestApi\Executor\Task\RestApiTaskExecutor $apiTaskExecutor
+     */
+    public function __construct(RestApiTaskExecutor $apiTaskExecutor)
+    {
+        $this->apiTaskExecutor = $apiTaskExecutor;
+    }
+
+    /**
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function helloWorld(): Response
```

Comment:
__invoke

Reply:
agree. As we already agreed to go with single `__invoke()` method, let's stick to the agreement. @sergeyspryker, please change this controller and `IndexController`